companies-manifest.js

```diff
@@ -817,6 +818,22 @@ const companies = [
     style: {},
     projects: {},
   },
+  {
+    key: 'https://www.welcometothejungle.com',
+    similarWebGlobalRank: 32472,
+    name: 'Welcome to the Jungle',
+    logo: WelcomeToTheJungleLogo,
+    projects: {
+      "welcome-ui": {
+        title: 'Welcome UI',
+        link: 'http://www.welcome-ui.com',
+        src: '/screenshots/thumbnails/welcome-ui.com.jpg',
```

Comment:
The actual screenshot's currently residing in the `screenshots` directory root. You are referencing the thumbnails directory though, thus resulting in the image not displaying correctly on the showcase. Please move the image to `thumbnails` :)